Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt

```diff
@@ -90,69 +76,12 @@ class TracingRepository @Inject constructor(
         it.taskState.isActive && it.taskState.request.type == EwRiskLevelTask::class
     }
 
-    fun refreshRiskResult() = scope.launch {
-        Timber.tag(TAG).d("refreshRiskResults()")
-
+    fun runRiskCalculations() = scope.launch {
+        Timber.tag(TAG).d("runRiskCalculations()")
         exposureWindowRiskWorkScheduler.runRiskTasksNow(sourceTag = TAG)
         presenceTracingRiskWorkScheduler.runRiskTaskNow(sourceTag = TAG)
     }
 
-    /**
-     * Launches the RetrieveDiagnosisKeysTransaction and RiskLevelTransaction in the viewModel scope
-     */
-    suspend fun refreshRiskLevel() {
-        // check if the network is enabled to make the server fetch
-        val isNetworkEnabled = networkStateProvider.networkState.first().isInternetAvailable
```

Comment:
where network is checked now ?

Reply:
It is not on purpose - that was part of the issue. It will fail with download but run the calculation to show the failed risk card.